ts/src/pro/ripio.ts

```diff
@@ -152,115 +105,111 @@ export default class ripio extends ripioRest {
         return message;
     }
 
-    async watchOrderBook (symbol: string, limit: Int = undefined, params = {}) {
-        /**
-         * @method
-         * @name ripio#watchOrderBook
-         * @description watches information on open orders with bid (buy) and ask (sell) prices, volumes and other data
-         * @param {string} symbol unified symbol of the market to fetch the order book for
-         * @param {int|undefined} limit the maximum amount of order book entries to return
-         * @param {object} params extra parameters specific to the ripio api endpoint
-         * @returns {object} A dictionary of [order book structures]{@link https://docs.ccxt.com/#/?id=order-book-structure} indexed by market symbols
-         */
+    async watchOrderBook (symbol = undefined, limit = undefined, params = {}) {
+        if (symbol === undefined) {
+            throw new ArgumentsRequired (this.id + ' watchOrderBook() requires a symbol argument');
+        }
         await this.loadMarkets ();
         const market = this.market (symbol);
-        symbol = market['symbol'];
-        const name = 'orderbook';
+        const pair = this.parseSymbol (symbol);
+        const name = 'orderbook/level_3@' + pair;
```

Comment:
I'd add this as option so users can also listen to orderbook/level_2@ if they wish